RELEASING.md

```diff
@@ -0,0 +1,157 @@
+- Release candidate releases: `v0.x.y-rc.z`
+
+### Repository setup
+
+Clone the repository: `git clone git@github.com:metal3-io/utility-images`
+
+or if using existing repository, make sure origin is set to the fork and
+upstream is set to `metal3-io`. Verify if your remote is set properly or not
+by using following command `git remote -v`.
+
+- Fetch the remote (`metal3-io`): `git fetch upstream`
+This makes sure that all the tags are accessible.
+
+### Creating Release Notes
+
+- Switch to the main branch: `git checkout main`
+
+- Create a new branch for the release notes**:
+  `git checkout -b release-notes-1.x.x origin/main`
+
+- Generate the release notes: `RELEASE_TAG=v1.x.x make release-notes`
+   - Replace `v1.x.x` with the new release tag you're creating.
+   - This command generates the release notes here
+     `releasenotes/<RELEASE_TAG>.md` .
+
+- Next step is to clean up the release note manually.
+   - If release is not a beta or release candidate, check for duplicates,
+     reverts, and incorrect classifications of PRs, and whatever release
+     creation tagged to be manually checked.
+   - For any superseded PRs (like same dependency uplifted multiple times, or
+     commit revertion) that provide no value to the release, move them to
+     Superseded section. This way the changes are acknowledged to be part of the
+     release, but not overwhelming the important changes contained by the
+     release.
+
+- Commit your changes, push the new branch and create a pull request:
+   - The commit and PR title should be 🚀 Release v1.x.y:
+      -`git commit -S -s -m ":rocket: Release v1.x.x"`
+      -`git push -u origin release-notes-1.x.x`
+   - Important! The commit should only contain the release notes file, nothing
+     else, otherwise automation will not work.
+
+- Ask maintainers and release team members to review your pull request.
+
+Once PR is merged following GitHub actions are triggered:
+
+- GitHub action `Create Release` runs following jobs:
+   - GitHub job `push_release_tags` will create and push the tags. This action
+     will also create release branch if its missing and release is `rc` or
+     minor.
+   - GitHub job `create draft release` creates draft release. Don't publish the
+     release until release tag is visible in. Running actions are visible on the
+     [Actions](https://github.com/metal3-io/utility-images/actions)
+     page, and draft release will be visible on top of the
+     [Releases](https://github.com/metal3-io/utility-images/releases).
+     If the release you're making is not a new major release, new minor release,
+     or a new patch release from the latest release branch, uncheck the box for
+     latest release. If it is a release candidate (RC) or a beta release,
+     tick pre-release box.
+   - GitHub jobs `build_keepalived` build release images with the
+     release tag, and push them to Quay. Make sure the release tags are visible in
+     Quay tags pages:
```

Comment:
This need to match single image wording from IPAM etc.

Reply:
release related documentation has been removed.